core/src/Modal/Modal.less

```diff
@@ -27,11 +29,7 @@
 .Modal {
   background-color: @zesty-white;
   border-radius: 4px;
-  display: flex;
-  flex-direction: column;
-  max-width: 50%;
```

Comment:
We do not want modals to be able to do a full screen take over hiding the fact it is loaded over the current application. While we may want to increase what the max-width value is, we do want to constrain it at some point. 

I would propose 80%. This will allow enough space to show larger, interactive type, modal experiences but no lose the context of this being a modal over a running application

Reply:
restored original styles and added width: fit-content which is supported by all major browser https://caniuse.com/?search=width%3A%20fit-content